src/routing-table/index.js

```diff
@@ -0,0 +1,391 @@
+'use strict'
+
+// @ts-ignore
+const KBuck = require('k-bucket')
+const { xor: uint8ArrayXor } = require('uint8arrays/xor')
+const GENERATED_PREFIXES = require('./generated-prefix-list.json')
```

Comment:
we should have a browser empty list? I know we do not recommend using the DHT in the browser, but there are users who run it. It also will be part of the bundle, if we not have a browser alternative

Reply:
If they use it in the browser and we've removed random walk, it probably shouldn't be an empty list otherwise peer discovery will not work properly.  We could always have a 'light' version that's like, 1/4 of the values or something?

Reply:
yeah, good point. Ok, I think we can start with a 1/4 of the list for now. These are the kind of things I wish JS testground had shipped. 

We will likely need to revisit, but we will need to revisit all browser story either way. So, yes, let's do 1/4 for now.

Reply:
Done!